api/src/main/java/com/thesurvey/api/service/AuthenticationService.java

```diff
@@ -57,6 +66,12 @@ public Authentication authenticate(Authentication authentication)
     @Transactional
     public UserResponseDto register(UserRegisterRequestDto userRegisterRequestDto) {
         User user = userRepository.save(userMapper.toUser(userRegisterRequestDto));
+        userCertificationRepository.save(UserCertification.builder()
+            .user(user)
+            .certificationType(
+                CertificationType.MOBILE_PHONE) // completed MOBILE_PHONE certification during the register process
```

Comment:
I think we need to remove `mobile phone` certification from our service. (Since it's required value)